src/components/Positions.jsx

```diff
@@ -0,0 +1,193 @@
+        const accountCDPs = CDPs[getWalletInfo().address];
+        for (const [cdpID, value] of Object.entries(accountCDPs)) {
+        if (value["state"] == "open") {
+            res.push({
+            id: cdpID,
+            liquidationPrice: (
+                (1.15 * value["debt"]) /
+                value["collateral"]
+            ).toFixed(4),
+            collateral: value["collateral"],
+            debt: value["debt"],
+            committed: value.hasOwnProperty("committed") ? value["committed"] : 0,
+            });
+        }
+        }
+    }
+    if (res.length == 0) {
+        res = dummyCDPs;
+    }
+    return res;
+}
+const dummyCDPs = [
+    {
+      id: "N/A",
+      liquidationPrice: 0,
+      collateral: 0,
+      debt: 0,
+    },
+  ];
+  
+
+export default function Positions() {
+    var details = [
+        {
+            title: "Collateral",
+            val: `${0.00}%`,
+            hasToolTip: true,
+        },
+        {
+            title: "Received DAI",
+            val: `${0.00}%`,
+            hasToolTip: true,
+        },
+        {
+            title: "Liquidation Price",
+            val: `${0.00}%`,
+            hasToolTip: true,
+        },
+        {
+            title: "Received DAI",
+            val: `${0.00}%`,
+            hasToolTip: true,
+        },
+        {
+            title: "ETH exposure",
+            val: `${0.00}%`,
+            hasToolTip: true,
+        },
+        {
+            title: "Stability Fee",
+            val: `${0.00}%`,
+            hasToolTip: true,
+        },
+        {
+            title: "Liquidation ratio",
+            val: `${0.00}%`,
+            hasToolTip: true,
+        },
+        {
+            title: "DAI avaible from ETH",
+            val: `${0.00}%`,
+            hasToolTip: true,
+        },
+    ]
+    const {theme} = useContext(ThemeContext);
+    const [price, setPrice] = useState(0)
+    const loadedCDPs = CDPsToList();
+    const [currentCDP, setCurrentCDP] = useState(null)
+    useEffect(async () => {
+        setPrice(await getPrice());
+    }, []);
+    return <div>
+        <Header>
+            <b>Your Positions</b>
+            <b>Rewards</b>
+            <b>Total Balance</b>
+        </Header>
+        <Container>
+            {loadedCDPs.length && loadedCDPs.length > 0 ?
+                loadedCDPs.map((cdp) => {
+                    return (
+            <Position key={cdp.id}>
+                <PositionInfo>
+                    <div style={{display: "flex", flexDirection: "column", rowGap: 20}}>    
+                        <div>Supplied: ${(microalgosToAlgos(cdp.collateral) * price).toFixed(2)} in ALGOs</div>
+                        <div>Borrowed: ${mGardToGard(cdp.debt)} in GARD</div>
+                    </div>
+                    <div>APR 123%</div>
+                    <div style={{display: "flex", flexDirection: "column"}}>    
+                        <div>Health {`(100%)`}</div>
```

Comment:
Health should just be collateralization ratio, right?

Reply:
Makes sense, implemented.